Netable/Netable/Netable.swift

```diff
@@ -189,9 +233,16 @@ public actor Netable {
 
         let retryConfiguration = self.retryConfiguration
 
+        if let mocked = mockedUrl {
+            let data = try Data(contentsOf: mocked)
```

Comment:
Should we maybe be enforcing that mock URLs are file URLs in some way? I think right now you COULD get a remote URL here, and it would "work" but would be pretty weird (like could block the thread in some way, would make the URL request without the same headers as the normal request)

Reply:
I wonder if there are situations where we might want to let the mocked data come back from a server? some of the work Shayla has been doing with mocking involves grabbing shared info from Postman... @ssawchenko how helpful would this be?

Reply:
I was sort of assuming in the case where you were getting mocked data from a server that you would do it by rewriting the URLRequest, and that the mock case would mainly cover situations where you DIDN'T want to remote fetch it (i.e. the "you have.a JSON file in the app bundle").

Reply:
Oh of course, that makes sense. I'll look into making sure we only accept file urls.